test/lambda-grep.py

```diff
@@ -0,0 +1,85 @@
+import json
```

Comment:
I dropped this file in `/test` as it tests the Lambda function

Reply:
That sounds fine.